{"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError"]]},"identifier":{"url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","interfaceLanguage":"swift"},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)"]}],"kind":"symbol","abstract":[{"text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call.","type":"text"}],"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"cloudError","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"errorCode"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"?, "},{"kind":"externalParam","text":"errorMessage"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":"?)","kind":"text"}],"platforms":["Mac Catalyst"]}],"kind":"declarations"},{"kind":"parameters","parameters":[{"name":"errorCode","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"An optional error code as a string, that helps identify the type of error."}]}]},{"content":[{"inlineContent":[{"type":"text","text":"An optional human-readable message providing more details about the error."}],"type":"paragraph"}],"name":"errorMessage"}]}],"metadata":{"roleHeading":"Case","title":"NfinityCloudError.cloudError(errorCode:errorMessage:)","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"cloudError"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"errorCode"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?, "},{"kind":"externalParam","text":"errorMessage"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?)","kind":"text"}],"role":"symbol","symbolKind":"case","externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO05cloudF0yACSSSg_AEtcACmF","modules":[{"name":"PasswordAndNfinityFramework"}]},"schemaVersion":{"minor":3,"major":0,"patch":0},"references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudError(errorCode:errorMessage:)":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","title":"NfinityCloudError.cloudError(errorCode:errorMessage:)","abstract":[{"text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call.","type":"text"}],"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"cloudError"},{"kind":"text","text":"("},{"kind":"externalParam","text":"errorCode"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?, "},{"text":"errorMessage","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?)","kind":"text"}],"kind":"symbol","type":"topic"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","abstract":[],"role":"collection","url":"\/documentation\/passwordandnfinityframework","type":"topic","title":"PasswordAndNfinityFramework","kind":"symbol"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"role":"symbol","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"NfinityCloudError","kind":"identifier"}],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","type":"topic","title":"NfinityCloudError","navigatorTitle":[{"text":"NfinityCloudError","kind":"identifier"}],"abstract":[{"text":"Represents an enum hierarchy for errors related to the NfinityÂ Cloud.","type":"text"}],"kind":"symbol"}}}